testing/vcs/test_vcs_lambert.py

```diff
@@ -0,0 +1,31 @@
+import vcs,cdms2
+import os,sys
+f = cdms2.open(os.path.join(vcs.sample_data, "clt.nc"))
+s = f("clt")
+x = vcs.init()
+x.setantialiasing(0)
+x.drawlogooff()
+x.setbgoutputdimensions(1200,1090,units="pixels")
+iso = x.createisofill()
+p=x.createprojection()
+p.type="lambert"
+#p.centralmeridian = -80
```

Comment:
I find them instructive. It took me a bit of time to figure out the correct spelling hyphenation etc...